### ISSUE TITLE
Clear does not work on number input field on state 'ng-invalid'

### ISSUE THREAD

**Issue by baddonkey** [angular/protractor#1583]
I'm struggeling calling a .clear() on a number input field. By calling .sendKeys('abc') the field state changes to 'ng-invalid', the following .clear() has no effect. The test below will fail on the expect. If the character string 'abc' is replaced with a valid number, the test will succeed.

Scenario:

```
'use strict';

describe('my app', function () {

    browser.get('index.html');

    describe('Clear on number input field', function () {

        var numberField = element(by.model('someNumber'));
        it('Calling clear on a number field on state ng-invalid', function () {
            numberField.sendKeys('abc').clear().sendKeys('50');
            expect(numberField.getAttribute('value')).toBe((50).toString());
        });
    });
});
```

HTML-Snippet:

```
<form name="NumberFieldForm" ng-controller="NumberFieldController">
    <div>
        <input type="number" data-ng-model="someNumber"/>
    </div>
</form>
```

Environment:
- Windows 8.1
- Protracor 1.4.0
- NodeJS 1.4.28
- Chrome 39.0.2171.71
- Firefox 33.1.1

**Comment by amwill1** [comment]
this is still an issue for me. I've tried the suggestions in this question and did

    elem.click();
    browser.sleep(1000);
    elem.clear().sendKeys(val);

but this solution only seems to work sometimes. often what happens is the input gets cleared but then fills in the previous value before sending the new value. i'm not sure what to do. i just updated to Protractor 3.0.0

**Comment by juliemr** [comment]
External issue has been fixed in ChromeDriver 2.26.
